app/workers/badge_achievements/send_email_notification_worker.rb

```diff
@@ -0,0 +1,12 @@
+module BadgeAchievements
+  class SendEmailNotificationWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform(badge_achievement_id)
+      badge_achievement = BadgeAchievement.find_by(id: badge_achievement_id)
```

Comment:
I know we decided not to have the jobs raise more meaningful errors instead of silently failing until the migration is done, but maybe we can at least switch them to using guard clauses? I find that easier to follow than `if` modifiers tucked on at the end of a potentially long expression. Something like:

```ruby
badge_achievement = BadgeAchievement.find_by(id: badge_achievement_id)
return unless badge_achievement

NotifyMailer.new_badge_email(badge_achievement).deliver_now
```

How do other people feel about this?

Reply:
+1 by me

Reply:
Sighhh I had this _exact_ thought when I was moving it too, I just didn't do it 🤦‍♂️good call!